tools/README.md

```diff
@@ -0,0 +1,75 @@
+# KTX: Tools
+
+Utilities for working with Ktx or generating code for use with LibGDX.
+
+### Why?
+
+Like the LibGDX `gdx-tools` library, this library contains tools for development rather than use in your game itself.
```

Comment:
I think it's worth linking `gdx-tools` somehow, but I'm not sure if it's fully documented anywhere. There are a couple of related pages on the official wiki (related to tools such as texture packer), but nothing that would give an overview of the library. Maybe [the repository](https://github.com/libgdx/libgdx/tree/master/extensions/gdx-tools)?